christfellowshipchurch/web-app-v2
# Add View All and remove See More button

### About
This PR removed the See More button and adds a View All button

### Test Instructions
1. Open Discover page > any category, you should not see the "See more" button, but instead a "View All" button

### Screenshots
<img width="1390" alt="Screen Shot 2022-10-19 at 2 31 12 PM" src="https://user-images.githubusercontent.com/46769629/196774973-91146eb1-10fb-43ee-9937-a910686ae3fc.png">

### Closes Tickets
[CFDP-2248](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2248)

### Related Tickets
N/A
